bentoml/deployment/sagemaker/__init__.py

```diff
@@ -339,17 +358,187 @@ def delete(self):
 
 
 # Deployment Service MVP Working-In-Progress
+def generate_temporary_sagemaker_content(archive_path):
+
+    with TempDirectory() as tempdir:
+        shutil.copytree(archive_path, tempdir)
+
+        with open(os.path.join(tempdir, "nginx.conf"), "w") as f:
+            f.write(DEFAULT_NGINX_CONFIG)
+        with open(os.path.join(tempdir, "wsgi.py"), "w") as f:
+            f.write(DEFAULT_WSGI_PY)
+        with open(os.path.join(tempdir, "serve"), "w") as f:
+            f.write(DEFAULT_SERVE_SCRIPT)
+
+        # permission 755 is required for entry script 'serve'
+        permission = "755"
+        octal_permission = int(permission, 8)
+        os.chmod(os.path.join(tempdir, "serve"), octal_permission)
+    return tempdir
+
+
 class SageMakerDeploymentOperator(DeploymentOperatorBase):
     def apply(self, deployment_pb):
-        # deploy code.....
+        deployment_spec = deployment_pb.spec
+        sagemaker_config = (
+            deployment_spec.deployment_operator_config.sagemaker_operator_config
+        )
+        if sagemaker_config is None:
+            raise BentoMLDeploymentException('Sagemaker configuration is missing.')
+
+        if sagemaker_config.region is not None:
+            region = sagemaker_config.region
+        else:
+            region = config.get('aws', 'default_region')
+
+        sagemaker_client = boto3.client('sagemaker', region)
+
+        repository = get_local()
+        bento = repository.get(deployment_spec.bento_name, deployment_spec.bento_version)
```

Comment:
@parano Is this is the way I get archive from repo? or I have store available to me?